<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>721</width>
    <height>621</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Geometry fitting</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="_imageHolder">
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,0,0,0,1,0">
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QGroupBox" name="groupBox_8">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string>Help</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_12">
         <item>
          <widget class="QLabel" name="label_48">
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Fit the modelization of the rings to your acquisition.&lt;/p&gt;&lt;p&gt;The reset button will use the previously picked rings to reach an aproximation of the geometry, while the fit fonction will tune current parameters to improve the result.&lt;/p&gt;&lt;p&gt;You can custom locks on the physical parameters to avoid the algorithm to change the values.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="_settings">
        <property name="title">
         <string>Experiment settings</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="_geometry">
        <property name="title">
         <string>Geometry</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="_fitting">
        <property name="title">
         <string>Fitting</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_10" columnstretch="0,1">
         <item row="1" column="0" colspan="2">
          <widget class="WaitingPushButton" name="_resetButton">
           <property name="text">
            <string>Reset</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0" colspan="2">
          <widget class="WaitingPushButton" name="_fitButton">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>40</height>
            </size>
           </property>
           <property name="text">
            <string>Fit</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="_currentResidual">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Residual:</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="_nextStep">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string>Next &gt;</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>WaitingPushButton</class>
   <extends>QPushButton</extends>
   <header>silx.gui.widgets.WaitingPushButton</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
